### PR TITLE
Adding fields to schema and dev version release

### PR DESCRIPTION
**Pull request by rachelannelise**
Adding the following fields to the schema:
- channel
- pageview_id
- pageload_id
- videostart_id

And cutting a new release `2.3.0.dev0` for beta testing.

### CODE REVIEW

**Comment by rachelannelise** [comment]
Looking through the failed checks - these look like old failures but will confirm.

File: parsely_raw_data/schema.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
     {"key": "action", "ex": "pageview", "type": str, "size": 256, "req": True},
     {"key": "apikey", "ex": "mashable.com", "type": str, "size": 256, "req": True},
     {"key": "campaign_id", "ex": "facebook_campaign", "type": str, "size": 256, "available_with_field": "url_query"},
+    {"key": "channel", "ex": "AMP", "type": str, "size": 64},
```
Comment (aldraco) [inline parsely_raw_data/schema.py]: nitpick: this will be lowercase, as per internal code (i.e. `amp`).

File: parsely_raw_data/schema.py [hunk 2]
```diff
@@ -65,6 +66,8 @@
     {"key": "metadata_thumb_url", "ex": "https://images.parsely.com/xY9xNBMulGDKRMzfKaUQzs7A9PA=/160x160/smart/http%3A//a.amz.mshcdn.com/media/ZgkyMDE2LzA5LzA3LzU2L0NyeFhpNjNYRUFBSnZwRS5lNDAyMy5qcGcKcAl0aHVtYgkxMjAweDYzMAplCWpwZw/156d0173/3ae/CrxXi63XEAAJvpE.jpg", "type": str, "available_with_field": "metadata"},
     {"key": "metadata_title", "ex": "Everyone has the same fear about Apple's new earbuds", "type": str, "available_with_field": "metadata"},
     {"key": "metadata_urls", "ex": ["http://mashable.com/2016/09/07/airpods-jokes/"], "type": list, "available_with_field": "metadata"},
+    {"key": "pageload_id", "ex": "66554433", "type": int},
```
Comment (aldraco) [inline parsely_raw_data/schema.py]: Can these 3 things be the same? (videostart, pageload, pageview ids)

Comment (rachelannelise) [reply parsely_raw_data/schema.py]: they could - but it would be unlikely - good catch I'll change them

File: parsely_raw_data/schema.py (hunk 1 above)
Comment (rachelannelise) [reply parsely_raw_data/schema.py]: +1 , I was looking at Dash

**Comment by rachelannelise** [comment]
Moving the Travis failures that are not related to these code changes to an issue: https://github.com/Parsely/parsely_raw_data/issues/32

**Pull request merged**